scripts/setupvars/setupvars.bat

```diff
@@ -40,16 +40,9 @@ set "PATH=%INTEL_OPENVINO_DIR%\deployment_tools\model_optimizer;%PATH%"
 :: Inference Engine
 set "InferenceEngine_DIR=%INTEL_OPENVINO_DIR%\deployment_tools\inference_engine\share"
 set "HDDL_INSTALL_DIR=%INTEL_OPENVINO_DIR%\deployment_tools\inference_engine\external\hddl"
-set "OPENMP_DIR=%INTEL_OPENVINO_DIR%\deployment_tools\inference_engine\external\omp\lib"
```

Comment:
Seems like OMP build option is no more relevant. Should we remove references to OMP? 

- https://github.com/openvinotoolkit/openvino/wiki/CMakeOptionsForCustomCompilation#options-affecting-binary-size
- https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/cmake/dependencies.cmake#L98

Reply:
removed